.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,13 @@
+**Reproduction**: [describe how you are able to reproduce ("trigger") this bug/issue.]
+
+**Expected behavior**: [describe the behavior you would expect the repro to yield.]
+
+**Actual behavior**: [describe the actual behavior, which is presented through the repro.].
+
+**Build information**: [output of `rustc -V`, `git rev-parse HEAD`, `qemu-i386 -version`, `uname -a`, etc.]
+
+**Misc**: [...]
```

Comment:
[optional: for other relevant information that should be known or cannot be describe in the other fields]